src/modeler/AutoRest.Swagger/Model/ServiceDefinition.cs

```diff
@@ -90,6 +90,7 @@ public ServiceDefinition()
         [CollectionRule(typeof(BodyTopLevelProperties))]
         [CollectionRule(typeof(HttpVerbValidation))]
         [CollectionRule(typeof(DeleteMustHaveEmptyRequestBody))]
+        [CollectionRule(typeof(PropertiesNamesCamelCase))]
```

Comment:
Shouldn't we traverse all definitions and apply this rule over all model definitions in the doc